app/views/stations/new.erb

```diff
@@ -4,19 +4,19 @@
   <div class="row">
     <div class="col-md-6 mb-1">
       <label for="validationCustom01">Name: </label>
-      <input type="text" class="form-control" name= "station[name]" value="Name of Station"/>
+      <input type="text" class="form-control" name= "station[name]" placeholder="Name of Station"/>
     </div>
     <div class="col-md-6 mb-1">
       <label for="validationCustom02">Dock Count: </label>
-      <input type="text" class="form-control" name="station[dock_count]" value="Number of Bikes at this Station"/>
+      <input type="text" class="form-control" name="station[dock_count]" placeholder="Number of Bikes at this Station"/>
```

Comment:
I bet these are the merge conflicts but I'll take care of all of that. Great work!